IPython/core/magics/script.py

```diff
@@ -54,6 +54,12 @@ def script_args(f):
             This is used only when --bg option is given.
             """
         ),
+        magic_arguments.argument(
+            '--raise_error', action="store_true",
```

Comment:
`--raise-error` (dash instead of underscore) seem more common for flags ?

Reply:
That seems reasonable